app/discovery/provider/docker.go

```diff
@@ -161,104 +151,99 @@ func (d *Docker) matchedPort(c containerInfo) (port int, err error) {
 	return port, nil
 }
 
-// activate starts blocking listener for all docker events
-// filters everything except "container" type, detects stop/start events and publishes signals to eventsCh
-func (d *Docker) events(ctx context.Context, client DockerClient, eventsCh chan discovery.ProviderID) error {
-	dockerEventsCh := make(chan *dc.APIEvents)
-	err := client.AddEventListenerWithOptions(dc.EventsOptions{
-		Filters: map[string][]string{"type": {"container"}, "event": {"start", "die", "destroy", "restart", "pause"}}},
-		dockerEventsCh)
-	if err != nil {
-		return fmt.Errorf("can't add even listener: %w", err)
-	}
+// changed in tests
+var dockerPollingInterval = time.Second * 10
```

Comment:
is this package level var should be package level? can we have a field? And probably we can even make it public and allow to set whatever caller needs

Reply:
Good point. Fixed